### PR TITLE
Stretch Card inside Grid (ignore branch name)

### PR DESCRIPTION
**Pull request by amimagid**
Added ability to stretch cards to the 100 percent of the container, and, added support for auto adjustable stretched cards

### CODE REVIEW

**Comment by nirhart** [comment]
I think it should be on the row, like .row-eq-height in [here](http://getbootstrap.com.vn/examples/equal-height-columns/)

**Comment by nirhart** [comment]
@amimagid ?

**Comment by amimagid** [comment]
@nirhart - Sorry, had no time to look into it, but I think that this solution is only for Bootstrap's rows, while what I suggested is for any kind of container. If the intention was for Card to be only inside a row then I agree we should stick to the Bootstrap guidelines and classes. What say you?

File: src/Grid/Grid.scss [hunk 1]
```diff
@@ -20,7 +20,9 @@ $grid-row-margin-bottom: 30px;
 		margin-bottom: $grid-row-margin-bottom;
 	}
 }
-
+.flex-container {
+	display: flex;
```
Comment (Faradey27) [inline src/Grid/Grid.scss]: wrong number of spaces?

File: stories/GridWithCardLayout/ExampleGrid.js [hunk 2]
```diff
@@ -28,19 +28,21 @@ export default () =>
   <div data-hook="card-example" className={styles.exampleContainer}>
     <Container>
       <AutoAdjustedRow>
-        <Card>
+        <Card stretchVertically={true}>
```
Comment (Faradey27) [inline stories/GridWithCardLayout/ExampleGrid.js]: why not just <Card stretchVertically> ?

Comment (nirhart) [inline stories/GridWithCardLayout/ExampleGrid.js]: just what?

Comment (Faradey27) [inline stories/GridWithCardLayout/ExampleGrid.js]: sorry) forgot quotes for code block

`<Card stretchVertically>`
...

Comment (nirhart) [inline stories/GridWithCardLayout/ExampleGrid.js]: ohh right, sure. @amimagid

**Pull request merged**